server/src/main/resources/db/mysql/migration/V14__secrets_owner.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE secrets ADD owner BIGINT;
+ALTER TABLE secrets ADD CONSTRAINT fk_secrets_owner_to_groups_id FOREIGN KEY (owner) REFERENCES groups (id);
```

Comment:
We don't typically define foreign key constraints in our migration scripts, but it feels like the right thing to do.